VoxelSniperBukkit/src/main/resources/plugin.yml

```diff
@@ -1,10 +1,10 @@
-name: ${project.artifactId}
+name: ${projectName}
 main: com.thevoxelbox.voxelsniper.bukkit.BukkitVoxelSniper
-version: ${project.build.version}
+version: ${version}
 api-version: 1.15
-website: ${project.url}
+website: http://www.voxelwiki.com/minecraft/VoxelSniper
```

Comment:
Might as well remove this/set to github since this website is not functional

Reply:
good point

Reply:
https://github.com/KevinDaGame/VoxelSniper-Reimagined/pull/51/commits/720e628c562c912ab9f5209d65c2e6512cfe6899